### PR TITLE
Feature/contact form milestone

### PR DESCRIPTION
**Pull request by Kgomotso196**
## PROJECT TO DO'S
**IN THIS PROJECT:**

> I collected data using the formspree.

> Built a contact form to present the footer section.

> There are no functionalities of JavaScripts.

> I used fields that strictly use required fields ("email' field requires exactly an email input) .

> I used Flexbox to correctly layout elements used.

### CODE REVIEW

**Review by Felix45 — CHANGES_REQUESTED** [review]
Hi @Kgomotso196 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights :heavy_check_mark: 
- [x] No linter errors :white_check_mark: 
- [x] GitHub flow is used correctly :white_check_mark: 
- [x] Good descriptive  commit message :white_check_mark: 
- [x] Descriptive pull request title and summary :white_check_mark:  


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -238,6 +238,59 @@ <h3 class="language-container-text">Skills</h3>
           to chat please get in touch.
         </p>
       </div>
-    </section>
+      <form action="https://formspree.io/f/xaykrgow" method="post" id="form">
```
Comment (GedeonTS) [inline index.html:241]: Good doing more than required for this milestone :clap: , However, Kindly make sure the form is a flexbox as requested by the previous reviewer. :recycle:

File: otherCSSfiles/bottom.css [hunk 2]
```diff
@@ -2,17 +2,140 @@
...
+form textarea {
+  width: 327px;
+  height: 100px;
+  border-radius: 4px;
+  border: 1px solid #60c095;
+  margin-top: 24px;
+  background: #fff;
+}
+
+.ftextarea {
```
Comment (GedeonTS) [inline otherCSSfiles/bottom.css:75]: - As requested by the previous reviewer [here](https://github.com/Kgomotso196/My-Portfolio/pull/4#discussion_r1116751800) , Kindly make sure you have the background images shown on the Figma design as it is a requirement for this project :recycle: 

As we can see below, your contact section doesn't have a background image:
Screenshot from 2023-02-24 13-03-24

**Review by GedeonTS — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED :recycle: :
Hi @Kgomotso196 , 

Great work on making some more changes to improve your project 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights :tophat: :+1: 
- No linter errors :+1: 
- Good use of GitHub flow :+1: 
 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions :ringed_planet: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by IngDavidTM — APPROVED** [review]
Hi @Kgomotso196  🙋‍♂️, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request follows the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 


## Optional suggestions

- [OPTIONAL] You did a wonderful job styling your project :+1:, but I think it would be better if you align the form to the center of the page because when the screen size grows the form remains on the left side of it

**Your design**

Screenshot from 2023-02-24 07-46-10


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**